stats_test.py

```diff
@@ -29,6 +29,6 @@ def test_range_():
 
 def test_standard_deviation():
     """Tests the SD function"""
-    assert stats.standard_deviation(A) == 2.0
-    assert stats.standard_deviation(B) == 7.07107
-    assert stats.standard_deviation(E) == 38.34058
+    assert stats.standard_deviation(A) == 1.63299
```

Comment:
So to clarify - it looks like statistics.stdev() finds the 'Sample Standard Deviation' and what your refactoring does is find the 'Population Standard Deviation' - get with your "client" and figure out what the actual "requirements" are